VERSION

```
@@ -1 +1 @@
-v4.2.7
+4.2.8
```

We dont need the v to be here, right?

---

Anyway, I'd to have this in now, for my next PR, if this is not good, we can fix it for 4.2.9

---

I don't think it'll break anything its only usage is in the CMake which strips the `v` and I think that manipulation should just silently do nothing.